packages/connections/src/stores/connections-store.ts

```diff
@@ -251,7 +256,11 @@ export function useConnections(
     ) {
       // Update lastUsed date as now and save the connection.
       connectedConnectionInfo.current.lastUsed = new Date();
-      void saveConnectionInfo(connectedConnectionInfo.current);
+      try {
+        void saveConnectionInfo(connectedConnectionInfo.current);
+      } catch (e) {
```

Comment:
this won't catch errors if `saveConnectionInfo` is a promise .. unless you remove void